docs/sdks/nodejs/storage.mdx

```diff
@@ -0,0 +1,108 @@
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleCodeBlock to SdkExampleCodeBlockImpl
+import { SdkExampleCodeBlockImpl } from "@site/src/components/SdkExampleCodeBlockImpl";
+
+# Getting Started with Momento Storage in JavaScript
+
+Momento provides two JavaScript SDKs; [one for Node.js](/sdks/nodejs) and [one for browsers other web applications](/sdks/web). The two SDKs have identical APIs, so your code will look the same except for `import` statements, but under the hood they are built for optimal performance and compatibility in different JavaScript runtime environments.
+
+This page contains the basics that you will need in order to get going quickly with Momento Storage. For more in-depth information and examples, visit the SDK pages linked above.
+
+## Install the Momento SDK
+
+To Install the Momento Node.js SDK in an existing Node.js project:
+
+```cli
+npm install @gomomento/sdk
+```
+
+OR, to install the Momento Web SDK in an existing browser application project:
+
+```cli
+npm install @gomomento/sdk-web
+```
+
+:::tip
+You only need one of the two libraries; either `@gomomento/sdk` or `@gomomento/sdk-web`, depending on your target platform. You do not need both.
+:::
+
+## Set up your API key
+
+You'll need a Momento API key to authenticate with Momento.  You can get one from the [Momento Web Console](https://console.gomomento.com/caches).
+Once you have your API key, store it in an environment variable so that the Momento client can consume it:
+
+```
+export MOMENTO_API_KEY=<your Momento API key here>
+```
+
+**Note**: it is best practice to put the API key into something like AWS Secret Manager or GCP Secret Manager instead of an environment variable for enhanced security, but we are using one here for demo purposes.
+
+## Import libraries and create a StorageClient object
+This code sets up the main function, pulls in the necessary imports, and instantiates the StorageClient that will be used to interact with your cache.
+
+<SdkExampleCodeBlock language={'javascript'} file={'examples/nodejs/storage/doc-example-files/cheat-sheet-main.ts'} />
+
+## Create a new store in Momento Storage
+Use this function to create a new store in your account.
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_CreateStore'} />
+
+## List the names of existing stores in your account
+A simple list of the names of stores for the account.
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_ListStores'} />
+
+## Write an item to a store
+A simple example of doing a put operation.
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_Put'} />
+
+## Read an item from a store
+This is an example of a simple read operation to get an item from a sore.
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_Put'} />
```

Comment:
it says Put even though you probably want to link to Get?

Reply:
thank you!

Reply:
fixed in https://github.com/momentohq/public-dev-docs/pull/1048/commits/3db326cf82cc07d18a23c34eb677316aeb3d119b